src/components/pages/Notes/NoteDialog/index.js

```diff
@@ -2,10 +2,11 @@ import React, { useState, useEffect } from 'react';
 import { Modal } from 'react-bootstrap';
 import useNotes from '../../../../contexts/NotesContext';
 import NoteDialogForm from './NoteDialogForm';
-import NoteDialogButtons from './NoteDialogButtons';
+import NoteDialogCreateFooter from './NoteDialogCreateFooter';
+import NoteDialogEditFooter from './NoteDialogEditFooter';
 
 const NoteDialog = () => {
-  const emptyNote = { title: '', content: '', variant: 0 };
+  const emptyNote = { title: '', content: '', variant: 0, color: '#ffc785' };
```

Comment:
What do you think about making default color of the note yellow?
![image](https://user-images.githubusercontent.com/51719931/79472886-ad10b800-8004-11ea-8127-93c3c975b1db.png)
```suggestion
  const emptyNote = { title: '', content: '', variant: 0, color: '#ffef7f' };
```